src/components/layout/navbar-items.tsx

```diff
@@ -19,35 +20,35 @@ export const NavbarItems: FC = () => {
         return '/'
     }
 
+    const currentDashboardUrl = dashboardURL()
+
+    // Define common hover style
+    const hoverStyle = { '--nl-hover': 'var(--mantine-color-blue-6)' }
+
     return (
         <Stack gap="xs">
-            <Button
-                fullWidth
-                className={styles.hover}
-                justify="flex-start"
-                variant="transparent"
+            <NavLink
+                label="Dashboard"
+                leftSection={<House />}
                 component={Link}
-                href={dashboardURL()}
+                href={currentDashboardUrl}
+                active={pathname === currentDashboardUrl}
                 c="white"
-                leftSection={<House />}
-            >
-                Dashboard
-            </Button>
+                color="blue.7"
+                variant="filled"
+                style={hoverStyle}
+            />
 
-            <Button
-                fullWidth
-                className={styles.hover}
-                justify="flex-start"
-                variant="transparent"
+            <NavLink
+                label="Settings"
+                leftSection={<Gear />}
                 onClick={() => openUserProfile()}
```

Comment:
we will want to add the active prop here too just for styling (it seems everything is white on hover without it)
```active={pathname === currentDashboardUrl}```

Reply:
yeah good point - it is quite weird 🤔 css should be good 👍 thanks!!